### PR TITLE
2nd Blog for girls code Mk

### PR DESCRIPTION
**Pull request by gtl90**

### CODE REVIEW

**Comment by gtl90** [comment]
img path wrong
 white space added to headings to format correctly
added bullet list at top

**Comment by gtl90** [comment]
Just a little more glitter - going to miss RGSoC

File: blog/_posts/2018-09-17-Code-2-a-good-summer.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+Coaches:
+
+Neil - for all his knowledge and his fabulous git workshop
+Kalan - for his guidance especially on testing
+PJ and Rob for helping during girls code sessions.
+
+RGSoC Organisers : for setting up RGSoC meetups, organising tickets for conferences, all the admin work that goes into organising these summers. You Rock!!
+
+Anna
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-Code-2-a-good-summer.md]: Hi, 'Anna' is there for 2 times. Did you mean 'Ana Sofia'? And did you also mean 'Vaishali', as I don't any RGSoC organizer named Valisha. :)

File: blog/_posts/2018-09-17-Code-2-a-good-summer.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Bahmni Project specific - Thoughtworks: For their patience and time for letting us help out on their project that they have a keen interest in.
+
+Angshu
+Konrad
+Sruti
+
+Thank you to our family for their unwavering support and putting up with our mood swings when code wasn’t working or we couldn’t work things out!
+
+![RGSoC Glitters](/img/blog/2018/RGSOC_glitters.jpg)
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-Code-2-a-good-summer.md]: The image seems to be broken for some reason. Please fix that.

File: blog/_posts/2018-09-17-Code-2-a-good-summer.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+---
+title: Team GirlsCode MK - Code 2 a good summer
+layout: post
+created_at: Sat Sep 15 2018
+permalink: blog/2018-07-25-Code-2-a-good-summer
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-Code-2-a-good-summer.md]: Hi @gtl90 ! Thanks for your PR. Lovely blog post. Please check the bunch of minor comments I have. 

Filename and permalink needs to be the same.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by gtl90** [comment]
I have updated the pull request not sure if this is correct way of doing or whether I should create a new pull request.  Let me know.  Thank you Vaishali :)

**Comment by v-thakkar** [comment]
That's the correct way @gtl90 ! Looks good to me now. Thanks!

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**